experiments/aldp/train.py

```diff
@@ -138,14 +138,14 @@
 
 # Set parameters for training
 ndim = 60
-loss_type = 'alpha_2_div' if 'loss_type' not in config['fab'] \
```

Comment:
@VincentStimper I'm not super confident about the changes I made to the Aldp scripts - could you please double check these?